docs/site/docs/develop/introduction.md

```diff
@@ -24,8 +24,10 @@ xmsg.sender          // who sent it (msg.sender of the source xcall)
 
 `omni` is a reference to the [OmniPortal](./contracts.md#omniportal) contract. One portal is deployed on each supported chain. To make interacting with the portal easier, inherit from the [XApp](./xapp/xapp.md) contract.
 
-:::info Only Finalized Cross-Chain Messages are Relayed
+:::info Omni is a Cross-Chain Messaging Protocol
 
-`XMsg` can only be relayed to the destination chain once the transaction it belongs to finalizes on Ethereum.
+Omni is a cross-chain messaging protocol that allows contracts to send and receive messages across different chains. It is a simple and secure way to interact with contracts on other chains.
+
+However, Omni is not a bridge for canonical or native tokens. If you need to move assets between chains, you will need to implement your own bridge (with a transfer token proxy and liquidity for source and destination rollups) or use an existing bridge.
```

Comment:
Should we also add that the bridge they build can use the Omni protocol, like our bridge built by Kevin does?